.golangci.yml

```diff
@@ -35,7 +35,18 @@ linters-settings:
   errorlint:
     errorf: false
   revive:
-    rules:
+    rules: # https://github.com/mgechev/revive/blob/master/RULES_DESCRIPTIONS.md
+      - name: atomic
+      - name: bare-return
+      - name: bool-literal-in-expr
+      - name: cognitive-complexity
+        arguments:
+          - 36 # TODO: gradually reduce it
+      - name: datarace
+      - name: duplicated-imports
+      - name: early-return
+      - name: empty-lines
```

Comment:
@savil If you don't like it, let's remove this.

Reply:
yeah, not a huge fan, lets remove please....

Reply:
Removed.